packages/env/src/index.ts

```diff
@@ -175,6 +175,7 @@ const smtpEnv = {
     SMTP_PORT: z.coerce.number().optional().default(25),
     SMTP_AUTH_DISABLED: boolean.optional().default("false"),
     SMTP_SECURE: boolean.optional().default("false"),
+    SMTP_IGNORE_TLS: boolean.optional().default("false"),
```

Comment:
```suggestion
    SMTP_IGNORE_TLS: boolean.optional(),
```

Because without it defined we want it to be `undefined` instead of `false` to match the previous behavior